tests/foreman/cli/test_contentviews.py

```diff
@@ -271,6 +271,74 @@ def test_cv_delete(self):
         self.assertNotEqual(result.return_code, 0)
         self.assertGreater(len(result.stderr), 0)
 
+    def test_delete_cv_version(self):
+        """@test: delete content view version
+
+        @feature: Content Views
+
+        @assert: content-view version deleted successfully
+
+        """
+
+        # Create new organization, product and repository
+        new_org = make_org({u'name': gen_alphanumeric()})
+        new_product = make_product({u'organization-id': new_org['id']})
+        new_repo = make_repository({u'product-id': new_product['id']})
+        # Sync REPO
+        result = Repository.synchronize({'id': new_repo['id']})
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
+        # Create new content-view and add repository to view
+        new_cv = make_content_view({u'organization-id': new_org['id']})
+        result = ContentView.add_repository({
+            u'id': new_cv['id'],
+            u'repository-id': new_repo['id'],
+            u'organization-id': new_org['id'],
+        })
+        self.assertEqual(
+            result.return_code, 0,
+            "Failed repository association: {0} and return code: {1}"
+            .format(result.stderr, result.return_code)
+        )
+        # Publish a version1 of CV
+        result = ContentView.publish({u'id': new_cv['id']})
+        self.assertEqual(
+            result.return_code, 0,
+            "Version1 publishing failed: {0} and return code: {1}"
```

Comment:
The message here could be as simple as 'Failed to publish version 1'. Other information is already available on logs.